<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericFunctionDialog</class>
 <widget class="QDialog" name="GenericFunctionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="recpgr_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>recpgr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dsphdr_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dsphdr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drdlat_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>drdlat_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cylrec_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cylrec_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="latsph_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>latsph_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cyllat_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cyllat_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="reclat_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>reclat_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dpgrdr_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dpgrdr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="recrad_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>recrad_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radrec_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>radrec_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sphrec_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>sphrec_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dgeodr_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dgeodr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="recsph_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>recsph_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pgrrec_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>pgrrec_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="lspcn_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>lspcn_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drdpgr_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>drdpgr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="subslr_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>subslr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="unload_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>unload_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="georec_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>georec_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drdcyl_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>drdcyl_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dcyldr_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dcyldr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sphcyl_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>sphcyl_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="latcyl_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>latcyl_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="latrec_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>latrec_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="recgeo_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>recgeo_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dlatdr_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>dlatdr_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drdsph_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>drdsph_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="xfmsta_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>xfmsta_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="furnsh_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>furnsh_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="drdgeo_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>drdgeo_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="subpnt_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>subpnt_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="reccyl_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>reccyl_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cylsph_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cylsph_</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sphlat_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>sphlat_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
